lib/av/commands/base.rb

```diff
@@ -180,6 +180,27 @@ def parse_param param
         end
         list
       end
+
+      def metadata_rotate degrees
+        add_output_param :'metadata:s:v:0', "rotate=#{degrees}"
+
+        self
+      end
+
+      def filter_metadata_rotate degrees
+        filter_rotate degrees
+
+        if @source
+          current_rotate = identify(@source)[:rotate] || 0
+        else
+          ::Av.log "Source has not been set - assuming current rotation metadata is 0"
+          current_rotate = 0
+        end
+
+        metadata_rotate (current_rotate - degrees) % 360
```

Comment:
I've not changed the rotation method (besides fixing the input/output params issue). 

My understanding is that filter_rotate rotates clockwise.

Metadata_rotate sets the metadata to whatever you tell it.

Filter_metadata_rotate rotates the image by the desired amount, then corrects the meta data. If the image is rotated 90 CW, then the metadata should show the recording device as having been rotate 90 CCW, ie 270